daemon/Cargo.toml

```diff
@@ -12,7 +12,6 @@ bdk = { version = "0.16", default-features = false, features = ["electrum"] }
 bdk-ext = { path = "../bdk-ext" }
 btsieve = { path = "../btsieve" }
 bytes = "1"
-cached = { version = "0.34.0", default-features = false, features = ["proc_macro"] }
```

Comment:
I would prefer if removing deserialisation cache was not part of this PR- I do not agree with the conclusion that slower startup performance is justified by removing well-tested ~20 lines of code.

It would be nice to see some data in isolation (along with data *how* much it degrades performance) in a different PR.

Reply:
This cache solved the same problem, just in a more hacky and less scalable way ...

The number of cache hits here will be at most 3 (because we have 4 different aggregates). For all closed CFDs, the number of cache hits will be 0 because we only deserialize them for projection. For the remainder of the program, this cache will just sit around and not do anything because the new aggregate cache will ensure we don't even load the events that are in this cache now.

The reason I want to remove this is because its uselessness is not clear unless you think this through really hard. Useless code that looks useful is dangerous and hinders evolving the codebase.

Reply:
There may be a possibility for more performance optimisation but those need to be re-evaluated in the context of the new functionality. The new performance is pretty good and I'd rather try to evaluate it from here again than keeping around old legacy!

Reply:
I created an issue: #1735